app/utils/dictionary/demography.ts

```diff
@@ -1035,10 +727,6 @@ export const countries = [
     name : 'Tuvalu',
     code : 'TV'
   },
-  {
-    name : 'Taiwan',
-    code : 'TW'
-  },
```

Comment:
Taiwan is an existing territory even though China does not recognize it. Where did you get the info for that list? I am not an expert on all countries but it seems not all data is correct here.